apps/apple/FluentUITester/macos/FluentUITester-macOS/ViewController.m

```diff
@@ -0,0 +1,21 @@
+#import "ViewController.h"
+#import "AppDelegate.h"
+
+#import <React/RCTRootView.h>
+
+@implementation ViewController
+
+- (void)loadView {
+  id<NSApplicationDelegate> appDelegate = [NSApp delegate];
+  if ([appDelegate isKindOfClass:[AppDelegate class]]) {
+    RCTBridge *bridge = [((AppDelegate *)appDelegate) bridge];
+    RCTRootView *rootView = [[RCTRootView alloc] initWithBridge:bridge moduleName:@"FluentUITester" initialProperties:nil];
+
+    [self setView:rootView];
+    [rootView addConstraint:[NSLayoutConstraint constraintWithItem:rootView attribute:NSLayoutAttributeWidth relatedBy:NSLayoutRelationEqual toItem:nil attribute:NSLayoutAttributeNotAnAttribute multiplier:1.0 constant:500]];

```

Comment:
recommended to use the anchor-based constraint APIs for more readability - [[rootView widthAnchor] constraintEqualToAnchor:500]. But should also split out 500 as a magic constant

Reply:
Actually a view shouldn't define its own width since this is the NSViewController's view

Reply:
Given the rootview has no superview, what view would the constraints be set on then?

Reply:
Maybe that's a reason to set this as a subview like it was originally